### DISCUSSION TITLE
Whether prefix url can be supported

### DISCUSSION THREAD

**Discussion by zhenruyan** [pocketbase/pocketbase#663] [Q&A]
Sometimes it is expensive to obtain the secondary domain name, such as the secondary domain name of some organizations, so it is hoped that the running instance can be isolated by matching the prefix url

#  give an example
 
```

./pockerbase  serve --prefix pocketbase1
./pockerbase  serve --prefix pocketbase2
./pockerbase  serve --prefix pocketbase3


```

demo.com/pocketbase1/_  
demo.com/pocketbase2/_
demo.com/pocketbase3/_

**Comment by ganigeorgiev** [comment]
I don't think this is possible out of the box because you can have only 1 application listening on a single port (in your case that will be 80/443).

If you want to host multiple applications on the same server listening on a single port, you can use a reverse proxy (eg. nginx, apache, caddy, etc.) and start each PocketBase instance on a different internal port (eg. 8091, 8092, 8093).

This currently is not very well documented, but you could find an example nginx config in https://pocketbase.io/docs/going-to-production/#deploy-behind-reverse-proxy.

**Reply by ganigeorgiev** [reply]
_Side note: There are multicast sockets and [`SO_REUSEPORT`](https://lwn.net/Articles/542629/) that seems to work for both tcp and udp and could be used to allow multiple applications binding to the same address and port, but in my opinion it is a lot easier and more flexible to have a reverse proxy managing your applications._

**Comment by fondoger** [comment]
Check my fork in case anyone needs this feature. https://github.com/fondoger/pocketbase/pull/12
